### PR TITLE
Fixed bug, cleaned up topic

### PR DESCRIPTION
**Pull request by hannanelson**
Bug fix is on line 204

### CODE REVIEW

File: modules/ROOT/pages/cloudhub/cloudhub-connector.adoc [hunk 1]
```diff
@@ -18,10 +17,9 @@ Exchange: https://www.mulesoft.com/exchange/com.mulesoft.connectors/mule-cloudhu
 
 The basic CloudHub configuration has the following properties:
 
-* Username: The user of the Anypoint Platform Account.
-* Password: The password of the given user.
-* Enviroment: Optional. Indicates which CloudHub environment to use. By default
-it's the one defined as the default in Anypoint Platform.
+Username:: The user of the Anypoint Platform account
+Password:: The password of the given user
+Environment:: The CloudHub environment to use, by default, the environment defined as the default in Anypoint Platform
```
Comment (valkyrie69) [inline modules/ROOT/pages/cloudhub/cloudhub-connector.adoc]:
```suggestion
Environment:: The CloudHub environment to use (by default this is the environment defined as the default in Anypoint Platform)
```

File: modules/ROOT/pages/cloudhub/cloudhub-connector.adoc [hunk 2]
```diff
@@ -169,39 +172,36 @@ be obtained from the `id` property of a <<notification-object,Notification objec
 [[listing-applications]]
 == List Applications
 
-With the connector you can retrieve all the available applications in the given environment.
-
-There is nothing required to configure and works out of the box.
+Using the connector, you can retrieve all available applications in the given environment. There is nothing required to configure.
```
Comment (valkyrie69) [inline modules/ROOT/pages/cloudhub/cloudhub-connector.adoc]:
```suggestion
Using the connector, you can retrieve all available applications in the given environment. No configuration is required. 
```

**Review by valkyrie69 — APPROVED** [review]

File: modules/ROOT/pages/cloudhub/cloudhub-connector.adoc [hunk 3]
```diff
@@ -8,8 +8,7 @@ Support Category: https://www.mulesoft.com/legal/versioning-back-support-policy#
 
 CloudHub Connector v1.0
```
Comment (kbostrom) [inline modules/ROOT/pages/cloudhub/cloudhub-connector.adoc:10]: Instructions for edit:

I have a small PR with perhaps too much churn for a peer review. Can you take a look?
https://github.com/mulesoft/docs-connectors/pull/615/files
Existing topic: https://docs.mulesoft.com/connectors/cloudhub/cloudhub-connector
Files to edit: Just the one
Type: Standard
Areas: Green
Whenever you get a chance.

File: modules/ROOT/pages/cloudhub/cloudhub-connector.adoc [hunk 4]
```diff
@@ -47,20 +45,16 @@ If you create your app outside of Studio, update your POM file as follows:
...
 Mule converts RELEASE to the latest version of the connector.
 
-== Choose an Environment
+== List Available Environments
 
-=== List Available Environments
-
-Once the username and password are configured correctly, users can
-retrieve the available environments for an easier configuration.
+Once the username and password are configured correctly, you can
```
Comment (kbostrom) [inline modules/ROOT/pages/cloudhub/cloudhub-connector.adoc]: once > after, for internationalization

File: modules/ROOT/pages/cloudhub/cloudhub-connector.adoc [hunk 5]
```diff
@@ -47,20 +45,16 @@ If you create your app outside of Studio, update your POM file as follows:
...
 
 image::cloudhub/cloudhub-environment-value-provider.gif[Available Environments]
 
-=== Use the CloudHub Environment
+== Use the CloudHub Environment
 
-If the application is going to be used on CloudHub sometimes is desired to
-use the environment where the application is going to be deployed, for this case
-the user can use the <<environment.id-sysprop,`environment.id`>> system property.
+To specify the environment to which to deploy the app, use the <<environment.id-sysprop,`environment.id`>> system property.
```
Comment (kbostrom) [inline modules/ROOT/pages/cloudhub/cloudhub-connector.adoc]:
```suggestion
To specify the app deployment environment, use the <<environment.id-sysprop,`environment.id`>> system property.
```

File: modules/ROOT/pages/cloudhub/cloudhub-connector.adoc [hunk 6]
```diff
@@ -104,33 +102,38 @@ Optionally can be configured:
...
 
-Optionally can be configured:
+You can also configure:
 
-* Limit - The number of notifications to retrieve, by default it's -1, which means all available.
-* Status - Filters the notifications to only retrieve those that are unread, read, or all. By default only retrieves the ones marked as unread.
-* Search - Only retrieves the ones that contains the given text.
+* Limit
++
+Specifies the number of notifications to retrieve, by default `-1`, which indicates all available notifications
```
Comment (kbostrom) [inline modules/ROOT/pages/cloudhub/cloudhub-connector.adoc]:
```suggestion
Specifies the number of notifications to retrieve (default `-1`, which indicates all available notifications)
```

File: modules/ROOT/pages/cloudhub/cloudhub-connector.adoc [hunk 7]
```diff
@@ -104,33 +102,38 @@ Optionally can be configured:
...
+* Status
++
+Filters the notifications to retrieve (unread, read, or all), by default, only retrieves notifications marked as unread
```
Comment (kbostrom) [inline modules/ROOT/pages/cloudhub/cloudhub-connector.adoc]:
```suggestion
Filters the notifications to retrieve as unread, read, or all (default `unread`, which retrieves only notifications marked as unread
```

File: modules/ROOT/pages/cloudhub/cloudhub-connector.adoc [hunk 8]
```diff
@@ -149,13 +152,13 @@ This operation returns a list of notifications with the following structure.
...
 
 [[marking-notification]]
-== Mark Notification as Read
+== Mark a Notification as Read
+
+Use the Mark Notification operation to mark the notification as read. 
 
-Once a notification is consumed, marks the notification as Read
-so that the next time notifications are read, only the unread ones are retrieved.
+Once a notification is consumed, mark a notification as read so that the next time notifications are read, only the unread ones are retrieved.
```
Comment (kbostrom) [inline modules/ROOT/pages/cloudhub/cloudhub-connector.adoc]: ditto re. "once"; GLOBAL consideration

File: modules/ROOT/pages/cloudhub/cloudhub-connector.adoc [hunk 9]
```diff
@@ -149,13 +152,13 @@ This operation returns a list of notifications with the following structure.
...
 
-The operation Mark Notification only requires of the Notification ID, this can
-be obtained from the `id` property of a <<notification-object,Notification object>>.
+The Mark Notification operation requires only the notification ID, which you can obtain from the `id` property of a <<notification-object,Notification object>>.
 
 .Example of Marking notifications as Read
```
Comment (kbostrom) [inline modules/ROOT/pages/cloudhub/cloudhub-connector.adoc]: Do you intend title-style caps for this caption?

**Review by kbostrom — APPROVED** [review]
Easy peasy!
You write so clearly!

**Pull request merged**